container/docker/Dockerfile.alpine

```diff
@@ -0,0 +1,19 @@
+FROM alpine:latest
+
+RUN apk add nodejs
+RUN apk add npm
```

Comment:
Do we need this given our use of `gradle-node`?

Reply:
this wont be required as the regular build step works in image building. earlier that was not working.